### PR TITLE
Update resnet_model.py

### PR DESCRIPTION
**Pull request by ngujdi**
tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=self.labels) 
===> 
tf.nn.softmax_cross_entropy_with_logits(logits=self.predictions, labels=self.labels)

### CODE REVIEW

**Comment by ngujdi** [comment]
close
